mobile_back_samsung/samsung/lib/public/include/mbe_config_1200.pbtxt

```diff
@@ -15,7 +15,7 @@ benchmark_setting {
   framework: "ENN"
   delegate_choice: {
     delegate_name: "ENN_NPU"
-    accelerator_name: "npu"
+    accelerator_name: "samsung_npu"
```

Comment:
@Mostelk @AhmedTElthakeb 
Almost all settings used `samsung_npu` as accelerator_name, and only some use `npu`, so I renamed `npu` to `samsung_npu` to make it uniform. Please confirm this change.

Reply:
@AhmedTElthakeb confirmed the change in the meeting today.